pkg/pillar/dpcreconciler/genericitems/dhcpcd.go

```diff
@@ -281,12 +288,12 @@ func (c *DhcpcdConfigurator) DhcpcdArgs(config types.DhcpConfig) (op string, arg
 	switch config.Dhcp {
 	case types.DhcpTypeClient:
 		op = "--request"
-		args = []string{"-f", "/dhcpcd.conf", "--noipv4ll", "-b", "-t", "0"}
+		args = []string{"-f", "/etc/dhcpcd.conf", "--noipv4ll", "-b", "-t", "0"}
 		switch config.Type {
 		case types.NetworkTypeIpv4Only:
-			args = []string{"-f", "/dhcpcd.conf", "--noipv4ll", "--ipv4only", "-b", "-t", "0"}
+			args = []string{"-f", "/etc/dhcpcd.conf", "--noipv4ll", "--ipv4only", "-b", "-t", "0"}
```

Comment:
new version keeps it in a different folder? When upgrading we discard the previous version?